package.json

```diff
@@ -11,8 +11,12 @@
     "Guillaume Renard <grenard@nuxeo.com>"
   ],
   "license": "Apache 2.0",
+  "engines": {
+    "node": ">=8.0.0"
+  },
   "private": true,
   "scripts": {
+    "postinstall": "check-engine || true",
```

Comment:
What does the `true` do?

Reply:
Whenever we're using a version of `check-engine` that does not support the `--ignore` flag we have to do this little "trick" to prevent the check from returning an error code. We just want to log the info. `|| true` here ensures that this script will always exit with `0`.